Flutter/datagrid/editing.md

```diff
@@ -564,6 +564,90 @@ Widget build(BuildContext context) {
 {% endhighlight %}
 {% endtabs %}
 
+## How to check whether current cell is in editing mode
+
+You can check whether current cell is in editing or not from the `SfDataGrid.isCurrentCellInEditing`. If current cell is in edit mode it will return true else return false.
```

Comment:
You can check whether current cell is in editing or not by using  `DataGridController.isCurrentCellInEditing` property.